nntrainer/layers/conv2d_layer.h

```diff
@@ -35,7 +35,9 @@ class Conv2DLayer : public LayerImpl {
   Conv2DLayer(unsigned int filter_size_ = 0,
               const std::array<unsigned int, CONV2D_DIM> &kernel_size_ = {0, 0},
               const std::array<unsigned int, CONV2D_DIM> &stride_ = {1, 1},
-              const std::array<unsigned int, CONV2D_DIM * 2> &padding_ = {0, 0, 0, 0}) :
+              const std::array<unsigned int, CONV2D_DIM * 2> &padding_ = {0, 0,
+                                                                          0,
+                                                                          0}) :
```

Comment:
is it right indentation?

Reply:
I find this strange too but clang-format-6.0 updates to this indentation.

Reply:
alright.